### PR TITLE
Update CMake target fot xz_utils

### PR DESCRIPTION
**Pull request by uilianries**
Specify library name and version:  **xz_utils/5.2.4**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by gocarlos** [comment]
@danimtb if more and more PR take 8 days like this to merge, then the CCI will not work because it will be a bottle neck for the whole cpp community...

something that can be changed to merge PR quicker?

**Comment by danimtb** [comment]
@gocarlos remember this project is still in EAP and there are still many things to improve. We are working hard to improve the building service and we will automate all the processes as much as possible in the future.

Meanwhile, we still have to do manual revisions and merges for the correct operation of the service. I hope this makes sense. Thanks a lot for the feedback! 😄

File: recipes/xz_utils/all/conanfile.py [hunk 1]
```diff
@@ -133,3 +133,5 @@ def package_info(self):
             self.cpp_info.defines.append("LZMA_API_STATIC")
         self.cpp_info.libs = tools.collect_libs(self)
         self.cpp_info.names["pkg_config"] = "liblzma"
+        self.cpp_info.names["cmake_find"] = "LibLZMA"
+        self.cpp_info.names["cmake_find_multi"] = "LibLZMA"
```
Comment (danimtb) [inline recipes/xz_utils/all/conanfile.py]:
```suggestion
        self.cpp_info.names["cmake_find_package"] = "LibLZMA"
        self.cpp_info.names["cmake_find_package_multi"] = "LibLZMA"
```

Comment (uilianries) [reply recipes/xz_utils/all/conanfile.py]: Sorry, I can't see the error. Could you point me? https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info According docs, cmake_find_package and cmake_find_package_multi are correct.

Comment (uilianries) [reply recipes/xz_utils/all/conanfile.py]: oh boy. forget about ...

**Comment by danimtb** [comment]
There was a connection timeout. Relaunching again!

**Pull request merged**